nss-day-cohort-46/rare-rest-rare-api-cool-cake
# Pp post reactions

Description:

Backend code for adding reactions to a post.

Testing Instructions:

Client side code:
```
git checkout main
git fetch --all
git checkout pp-postReaction
```

On the client side, add a reaction to a post. The counter should increment and the user should not be able to use the same reaction again.
